tests/integration/rules/length.phpt

```diff
@@ -85,14 +86,14 @@ try {
 ?>
 --EXPECT--
 "phpsp.org.br" must have a length lower than 5
-"phpsp.org.br" must have a length greater than 20
+"phpsp.org.br" must have a length greater than or equal to 20
```

Comment:
I think it would be better to keep both messages

Reply:
Indeed, I updated the pull request to include all messages.